src/Core/src/Eventuous.Subscriptions/Logging/Logger.cs

```diff
@@ -50,7 +50,6 @@ public class LogContext {
         WarnLog        = GetLogger(LogLevel.Warning);
         ErrorLog       = GetLogger(LogLevel.Error);
 
-        InternalLogger? GetLogger(LogLevel logLevel)
-            => Logger.IsEnabled(logLevel) ? new InternalLogger(Logger, logLevel, SubscriptionId) : null;
+        InternalLogger? GetLogger(LogLevel logLevel) => new InternalLogger(Logger, logLevel, SubscriptionId);
```

Comment:
Why did you remove the check for `IsEnabled`? Without it, the nullable return makes no sense, but, most importantly, the whole log thing will be called even is the given log level isn't enabled. The whole point of the internal logger is _not_ to do anything when low-level logs are produced but logging for that level isn't enabled.

Reply:
It's been a while since I made this change internally, but as I recall it I made it because when changing the loglevel during runtime (possible with fx Seq) the new loglevel was not picked up by the internallogger because it was not enabled when first called. 
I can see that it is not optimal, but I am unsure how else to fix it

Reply:
Well, the issue is that debug and trace logs are very verbose, and logging only allows avoiding interpolation, but not boxing. All the properties sent to the logger are `object`, so if there's any value type property - it is boxed. Which is exactly what is avoided by not calling the log at all using null for specific log level-based variables. If runtime log level change is a requirement, it should be handled differently somehow, but I would suggest making an issue just for that and create a separate PR to resolve it.